### DISCUSSION TITLE
Docker Container

### DISCUSSION THREAD

**Discussion by nickm324** [CorentinTh/it-tools#393] [General]
How often is the docker container updated, is it after each new feature(s) or on a set schedule.
Finding that when I update my container it doesn't have the most recent updates, even though I am pulling latest and it has been about a week since the last time I did a pull to update my local version and there has been multiple additions since the last pull.

**Comment by cgoIT** [comment]
See #304 and #305

**Reply by nickm324** [reply]
Thanks, that answers my question.
